internal/age/ssh.go

```diff
@@ -281,7 +284,18 @@ func NewSSHEd25519Identity(key ed25519.PrivateKey) (*SSHEd25519Identity, error)
 func ParseSSHIdentity(pemBytes []byte) (Identity, error) {
 	k, err := ssh.ParseRawPrivateKey(pemBytes)
 	if err != nil {
-		return nil, err
+		if !strings.Contains(err.Error(), "cannot decode encrypted private keys") {
+			return nil, err
+		}
+
+		fmt.Print("Enter passphrase for encrypted id_rsa: ")
```

Comment:
This message should use a user-recognizable key identifier, because you can encrypt non-RSA identities and there may be multiple SSH identities on the system. Can you pass the filename down to here, or return a detectable error that the caller can use to call a different version of the function?

Reply:
Since `age` is alpha alpha and the api is in flux, I don't see much value in mucking around with the internals.

Were I designing the api, I'd make it completely explicit.  And usage for rsa encrypt decrypt would not do any auto-detecting.

It would just be usage like:
```
echo "secret messages" | ./age "`curl -s https://github.com/eliwjones.keys`" > /tmp/secret.age
```
and:
```
./age -d -i /tmp/secret.age ~/.ssh/id_rsa
```

I guess one could make it more complex and have the encrypted file have information in the header on what public key to use for decryption.. if FiloSottile has plans for that, I'd imagine he will have opinions on how exactly he wants to handle encrypted `id_rsa` files.


This PR is really just for anyone whose mucking about and was wondering how to use their encrypted `id_rsa` with it.

Reply:
Ha, as is my wont.. I say "No", and then immediately make the change.

Was trivial to pass down the `name` argument and be explicit on what we are asking for the password for.